datahub-web-react/src/app/home/HomePageHeader.tsx

```diff
@@ -157,7 +156,7 @@ export const HomePageHeader = () => {
                 types: [],
                 query: '*',
                 start: 0,
-                count: 20,
```

Comment:
LOL we were fetching 20 then slicing down to 3? totally down for this change, but I guess the only functionality that we'd be losing is that we were randomly sorting and pulling 3 from this set of 20 so you'd get different suggestions each time you refresh/come back to the home page. Are you cool with just showing the same 3 every time (but in random order)? idk how we order search results from gms if it's guaranteed to be the same 3 every time or not.

All this being said I'm totally cool with only fetching 3 and showing the same 3 if need be. We could always fetch like 5 or 6 if we really wanted to mix it up (better than 20).

Reply:
Ha - yes! I'm okay to do 6 and shuffle between them